tests/chains/MultiversX/SerializationTests.cpp

```diff
@@ -28,7 +27,14 @@ TEST(MultiversXSerialization, SerializeTransactionWithData) {
     transaction.chainID = "1";
     transaction.version = 2;
 
-    string expected = (boost::format(R"({"nonce":42,"value":"1000000000000000000","receiver":"%1%","sender":"%2%","gasPrice":1000000000,"gasLimit":54500,"data":"Zm9v","chainID":"1","version":2})") % BOB_BECH32 % ALICE_BECH32).str();
+    string expected =
+        "{"
```

Comment:
In `tests/chains/MultiversX/SerializationTests.cpp` we test for these exact strings to be produced.